projects/js-packages/components/components/icon-tooltip/stories/index.tsx

```diff
@@ -51,6 +54,25 @@ const Template = args => (
 	</div>
 );
 
+const NewTemplate = args => (
+	<div style={ { position: 'absolute', height: '1000px', left: '300px', top: '300px' } }>
+		<div className="donut-meter-wrapper">
+			<DonutMeter segmentCount={ 100 } totalCount={ 500 } />
```

Comment:
I think it'd be best to avoid making the component story dependent on a separate, unrelated component.